src/main/java/com/bobocode/bibernate/configuration/AvailableSettings.java

```diff
@@ -0,0 +1,16 @@
+package com.bobocode.bibernate.configuration;
+
+public enum AvailableSettings {
```

Comment:
**(nit)** I would rename it to something like "PropertyNamesHolder" or "PropertyValues", or so

Reply:
I will rename to PropertyValues